cargo-apk/README.md

```diff
@@ -73,6 +73,26 @@ version = 1
 name = "android.permission.WRITE_EXTERNAL_STORAGE"
 max_sdk_version = 18
 
+# See https://developer.android.com/guide/topics/manifest/queries-element#provider
+[[package.metadata.android.queries.provider]]
+authorities = "org.khronos.openxr.runtime_broker;org.khronos.openxr.system_runtime_broker"
+# Note: The `name` attribute is not required for a queries provider, this is
+# a workaround for aapt throwing errors about missing `android:name` attribute.
```

Comment:
Nit:

```suggestion
# Note: The `name` attribute is normally not required for a queries provider, but is non-optional
# as a workaround for aapt throwing errors about missing `android:name` attribute.
# This will be made optional if/when cargo-apk migrates to aapt2.
```